File: Latihan2/latihanbc++/SepakBola.cpp
//Pembuatan kelas untuk tim sepak bola
class SepakBola{
	//Beberapa atribut tim sepak bola yang bersifat private
	private:
	string nama;
	string negara;
	string tahun;
	string pemain;
	string pendiri;
	string manajer;
	
	//constructor
	public:
	SepakBola(){
	}
	
	//Beberapa prosedur/fungsi menggunakan setter & getter
	
	void setNama(string nama){
		this->nama = nama;
	}
	
	string getNama(){
		return this->nama;
	}
	
	void setNegara(string negara){
		this->negara = negara;
	}
	
	string getNegara(){
		return this->negara;
	}
	
	void setTahun(string tahun){
		this->tahun = tahun;
	}
	
	string getTahun(){
		return this->tahun;
	}
	
	void setPemain(string pemain){
		this->pemain = pemain;
	}
	
	string getPemain(){
		return this->pemain;
	}
	
	void setPendiri(string pendiri){
		this->pendiri = pendiri;
	}
	
	string getPendiri(){
		return this->pendiri;
	}
	
	void setManajer(string manajer){
		this->manajer = manajer;
	}
	
	string getManajer(){
		return this->manajer;
	}
	
	//destructor
	~SepakBola(){
	}
};
	
	

File: Latihan2/latihanbphp/Daftartimspb/SepakBola.php
<?php 
	/* Buat program untuk membuat daftar tim sepakbola yang memiliki atribut
	nama tim, negara asal tim, tahun berdiri tim, serta pemain, menggunakan
	setiap bahasa (C++, PHP, Python dan Java)! */
 
 class SepakBola{

	private $nama;
	private $negara;
	private $tahun;
	private $pemain;
	private $pendiri;
	private $manajer;
		
	function __construct(){
	}
		
	function setNama($nama){
		$this->nama = $nama;
	}
		
	function getNama(){
		return $this->nama;
	}
		
	function setNegara($negara){
		$this->negara = $negara;
	}
		
	function getNegara(){
		return $this->negara;
	}
		
	function setTahun($tahun){
		$this->tahun = $tahun;
	}
		
	function getTahun(){
		return $this->tahun;
	}
		
	function setPemain($pemain){
		$this->pemain = $pemain;
	}
		
	function getPemain(){
		return $this->pemain;
	}
	
	function setPendiri($pendiri){
		$this->pendiri = $pendiri;
	}
		
	function getPendiri(){
		return $this->pendiri;
	}
	
	function setManajer($manajer){
		$this->manajer = $manajer;
	}
		
	function getManajer(){
		return $this->manajer;
	}
		
	 function __destruct(){
	 }
}
?>
